pdns/rec-lua-conf.cc

```diff
@@ -233,26 +234,34 @@ void loadRecursorLuaConfig(const std::string& fname)
 		    });
 
   Lua.writeFunction("addDS", [&lci](const std::string& who, const std::string& what) {
+      if(g_dnssecmode == DNSSECMode::Off)
+        theL()<<Logger::Warning<<"Warning: adding Trust Anchor for DNSSEC (addDS), but dnssec is set to 'off'!"<<endl;
```

Comment:
I see this code duplicated 3 more times with minor edits, maybe make a function?